compute/cloud-client/src/main/java/compute/disks/CreateSecondaryCustomDisk.java

```diff
@@ -102,9 +102,9 @@ public static Disk createDiskSecondaryCustom(String primaryProjectId, String sec
           .get(3, TimeUnit.MINUTES);
 
       if (response.hasError()) {
-        return null;
+        throw new Error("Error creating secondary custom disks! " + response.getError());
       }
-      return disksClient.get(secondaryProjectId, secondaryDiskZone, secondaryDiskName);
+      return response.getStatus();
```

Comment:
Return the operation response.

Reply:
Returned Operation.Status as was done for other samples